iOS_SDK/OneSignalSDK/OneSignalUser/Source/OSIdentityModel.swift

```diff
@@ -29,23 +29,7 @@ import Foundation
 import OneSignalCore
 import OneSignalOSCore
 
-// By matching the enum name to the raw value, it will always stringify correctly
-enum OSDefaultAlias: String {
-    // swiftlint:disable identifier_name
-    case onesignal_id = "onesignal_id"
-    case external_id = "external_id"
-    // swiftlint:enable identifier_name
-}
-
 class OSIdentityModel: OSModel {
-    /**
-     Set either `onesignal_id` or `external_id`, representing the alias that will be used in requests.
-     */
-    var primaryAliasLabel: OSDefaultAlias = .onesignal_id
```

Comment:
We will probably want to use this concept for JWT, since we will make requests with `external_id` when it is enabled. I am still ok with removing this code in this PR to if you want to, just making a note in-case this didn't come up.

Reply:
@jkasten2 I thought about JWT before removing this. The issue with the `primaryAliasLabel` as it exists here is that every Identity Model needs to manage it. Therefore, when the SDK first detects JWT is on, it needs to go and update every Identity Model when that flag can be centralized.